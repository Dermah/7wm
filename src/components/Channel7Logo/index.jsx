import * as React from "react";
import styled from "styled-components";

import Logo from "./logo.png";

const StyledLogo = styled.img`
  width: 2em;
  height: 2em;
`;

export default () => <StyledLogo alt="Channel 7 Logo" src={Logo} />;
