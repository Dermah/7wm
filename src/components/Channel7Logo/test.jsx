import * as React from "react";
import renderer from "react-test-renderer";

import Channel7Logo from "./";

describe("Channel7Logo", () => {
  test("snapshot test", () => {
    const tree = renderer.create(<Channel7Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
