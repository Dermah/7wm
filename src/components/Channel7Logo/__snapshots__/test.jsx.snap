// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Channel7Logo snapshot test 1`] = `
<img
  alt="Channel 7 Logo"
  className="sc-bdVaJa ldYOxM"
  src="logo.png"
/>
`;
